src/cache.ts

```diff
@@ -95,6 +97,7 @@ export class Cache {
     const response = await fetch(`https://${this.registry}/${qualified}`, {
       headers,
       agent: this.agent,
+      timeout: 10000,
```

Comment:
Had an occurrence where it looks like getMeta() calls were never ending, although not sure, feels weird
Anyway specifying a timeout here didn't seem like a bad idea